samples/tls/monitoring.md

```diff
@@ -2,114 +2,132 @@
+## Notes on Customization Values
 
-* Set `TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file to specify
+- Set `TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file to specify
 that Prometheus, Grafana, and Alertmanager are deployed with TLS enabled for
 connections to these components. Due to limitations in the underlying Helm
 charts, some components might not have TLS enabled for in-cluster connections
-between components. See the **Limitations and Known Issues** section below for
+between components. 
+See [Limitations and Known Issues](#Limitations-and-Known-Issues) for
 details.
 
-* Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace any
-sample hostnames with hostnames for your deployment. Specifically, you must
-replace `host.cluster.example.com` with the name of the ingress node. Often,
-the ingress node is the cluster master node, but your environment might be different.
+- Edit the `$USER_DIR/monitoring/user-values-prom-operator.yaml` file and replace any
+sample host names with the host names for your deployment. Specifically, you must
+replace `host.cluster.example.com` with the name of the ingress node. Typically,
+the ingress node is the cluster primary node.
 
-* If you are using ingress with TLS (HTTPS), you must manually populate these
+## Specifying TLS for Ingress
+
+If you are using ingress with TLS (HTTPS), you must manually create the following 
 Kubernetes secrets with TLS certificates before you deploy cluster monitoring:
 
-* `prometheus-ingress-tls-secret`
-* `alertmanager-ingress-tls-secret`
-* `grafana-ingress-tls-secret`
+- `prometheus-ingress-tls-secret`
+- `alertmanager-ingress-tls-secret`
+- `grafana-ingress-tls-secret`
+
+**Note:** The process of generating the TLS certificates for these secrets is outside the scope for this example.
 
-You can use the following command to generate the necessary secret for each set
-of TLS certificates:
+1. After you have obtained the TLS certificates for each secret, use the following command to generate the secrets. You must run the command for each secret name.
 
 ```bash
 kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
 ```
 
-Use `prometheus-ingress-tls-secret`, `alertmanager-ingress-tls-secret`, and
-`grafana-ingress-tls-secret` as values for `$SECRET_NAME`. Use `monitoring` for
-the value of `$NAMESPACE`.
+    where:
+
+- `$SECRET_NAME` is the one of the following values: 
+
+  - The first time use `prometheus-ingress-tls-secret`.
+  - The second time use `alertmanager-ingress-tls-secret`.
+  - The third time use `grafana-ingress-tls-secret`.
+
+- `$NAMESPACE` is the value `monitoring`.
+
+- `$CERT_KEY` is the TLS certificate key file associated with the secret name. Be sure to use the correct .key file in each instance of this command.
+
+- `$CERT_FILE` is the TLS certificate file associated with the secret name. Be sure to use the correct .crt file in each instance of this command.
```

Comment:
The naming convention for replaceable parts here should match the equivalent doc on the logging side.

Reply:
They now match:
<pre>
kubectl create secret tls <i>secret_name</i> --namespace <i>namespace</i> --key=<i>cert_key_file</i> --cert=<i>cert_file</i>
</pre>

  where:

- *secret_name* is the one of the following values:
  - The first time use `prometheus-ingress-tls-secret`.
  - The second time use `alertmanager-ingress-tls-secret`.
  - The third time use `grafana-ingress-tls-secret`.

- *namespace* is the value `monitoring`.
- *cert_key_file* is the TLS certificate key file associated with the secret name. 
  Be sure to use the correct .key file in each instance of this command.
- *cert_file* is the TLS certificate file associated with the secret name. 
  Be sure to use the correct .crt file in each instance of this command.